.github/workflows/preview.yml

```diff
@@ -11,9 +11,9 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       actions: read
-      # this should be enough to maintain comments as commenting on pull requests is done through an issues API
+      # needed to maintain comments
       issues: write
-      pull-requests: read
+      pull-requests: write
```

Comment:
Yeah, I was hoping we wouldn't need that as it's not very secure. What's odd is that it works sometimes...

Reply:
Yeah, I winced a little bit as I made that change, undoing all your nice security work ...